modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/user/profile/mgt/UserProfileAdminTestCase.java

```diff
@@ -65,8 +69,14 @@ public void testInit() throws Exception {
         
         userProfileMgtClient = new UserProfileMgtServiceClient(backendURL, sessionCookie);
         userMgtClient = new UserManagementClient(backendURL, sessionCookie);
-        
-        userMgtClient.addUser(userId1, "passWord1@", new String[]{"admin"}, "default");
+
+        ClaimValue[] claimValues = new ClaimValue[1];
+        ClaimValue lastName = new ClaimValue();
+        lastName.setClaimURI(lastNameClaimURI);
+        lastName.setValue(userId1);
+        claimValues[0] = lastName;
+
+        userMgtClient.addUser(userId1, "passWord1@", new String[]{"admin"}, "default", claimValues);
```

Comment:
Shall we initialize the `ClaimValue` array here itself if we are not using it anywhere below? In the same way that we have initialized the String array. It will be more readable and we can reduce the space IMO. Check the same in other classes. 

`userMgtClient.addUser(userId1, "passWord1@", new String[]{"admin"}, "default", new ClaimValue[]{lastName});`

Reply:
Initilaized the ClaimValue[] array as the string array in this and similar places